content/blog/posts/2021-03-11-getting-started-with-serverless.md

```diff
@@ -0,0 +1,176 @@
+---
+authorId: chuckcarpenter
+categories:
+  - graphql
+  - javascript
+  - serverless
+date: '2021-03-11'
+slug: getting-started-with-serverless
+title: Getting Started with Serverless
+---
+
+While Ship Shape engineers are deep practioners of Javascript and are often working on web applications for our clients, we sometimes are also creating services for these applications to service functions for the app or provide the data layer and persistance in the form of an API.
+
+We've used a number of languages for these services, but usually default to our expertise in Javascript and work within the Node.js run-time. This works well and we then integrate into whatever deployment pipeline the client may have in place. Recently, we've started creating these services with a serverless cloud solution, rather than deploy to server infrastructure.
```

Comment:
```suggestion
We've used a number of languages for these services, but usually default to our expertise in JavaScript and work within the Node.js run-time. This works well and we then integrate into whatever deployment pipeline the client may have in place. Recently, we've started creating these services with a serverless cloud solution, rather than deploy to server infrastructure.
```